frontend/src/features/map_planning/components/Map.tsx

```diff
@@ -68,36 +24,37 @@ function useMapUpdates() {
  * You only have to make sure that every shape has the property "draggable" set to true.
  * Otherwise they cannot be moved.
  */
-export const Map = () => {
-  useInitializeMap();
-
+export const Map = ({ layers }: MapProps) => {
   const untrackedState = useMapStore((map) => map.untrackedState);
   const undo = useMapStore((map) => map.undo);
   const redo = useMapStore((map) => map.redo);
   const selectedLayer = useMapStore((state) => state.untrackedState.selectedLayer);
 
-  const getToolbarContent = (layerName: LayerName) => {
+  const getToolbarContent = (layerType: LayerType) => {
     const content = {
-      Base: { right: <div></div>, left: <div></div> },
-      Plant: { right: <PlantLayerRightToolbar />, left: <PlantLayerLeftToolbar /> },
-      Drawing: { right: <div></div>, left: <div></div> },
-      Dimension: { right: <div></div>, left: <div></div> },
-      Fertilization: { right: <div></div>, left: <div></div> },
-      Habitats: { right: <div></div>, left: <div></div> },
-      Hydrology: { right: <div></div>, left: <div></div> },
-      Infrastructure: { right: <div></div>, left: <div></div> },
-      Labels: { right: <div></div>, left: <div></div> },
-      Landscape: { right: <div></div>, left: <div></div> },
-      Paths: { right: <div></div>, left: <div></div> },
-      Shade: { right: <div></div>, left: <div></div> },
-      Soil: { right: <div></div>, left: <div></div> },
-      Terrain: { right: <div></div>, left: <div></div> },
-      Trees: { right: <div></div>, left: <div></div> },
-      Warnings: { right: <div></div>, left: <div></div> },
-      Winds: { right: <div></div>, left: <div></div> },
-      Zones: { right: <div></div>, left: <div></div> },
+      [LayerType.Base]: { right: <div></div>, left: <div></div> },
```

Comment:
Rebase: there already should be a toolbar.

Reply:
I don't understand your comment.
We needed to change this part because we changed the string `LayerName` to the enum `LayerType`.

Reply:
When I open `frontend/src/features/map_planning/routes/Map.tsx` in master there is a BaseLayerRightToolbar, which seems to be missing here. So I thought that there might be a problem due to renaming this file and later changes on master.

Reply:
Probably you just need to rebase (or merge recent master) to fix this problem.